src/Compilers/VisualBasic/Portable/CodeGen/EmitConversion.vb

```diff
@@ -211,8 +212,18 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.CodeGen
                         ' But first Pop off the Null reference cause we don't need it anymore.
                         _builder.EmitOpCode(ILOpCode.Pop)
 
+                        Dim constructor = GetParameterlessValueTypeConstructor(DirectCast(typeTo, NamedTypeSymbol))
+
                         'TODO: used
-                        EmitLoadDefaultValueOfTypeFromConstructorCall(conversion.ConstructorOpt, used:=True, syntaxNode:=conversion.Syntax)
+                        If constructor Is Nothing OrElse constructor.IsDefaultValueTypeConstructor() Then
+                            EmitInitObj(typeTo, used:=True, syntaxNode:=conversion.Syntax)
```

Comment:
This is preexisting behavior (perhaps compat?), but it seems strange. "Nothing" should just map to initobj, there is no need to call constructor.
No need to deal with that in this change, just noticing...